.gitignore

```diff
@@ -33,5 +33,4 @@ config/environments/*.local.yml
 /public/assets/
 
 # Ignore any docker-compose created volume data
-/postgres-data/
```

Comment:
why is this being removed?  Does this PR need it?

Reply:
It's associated with the addition of the `postgres-data` volume in `docker-compose.yml`. Now this directory won't be created in the `dlme` directory; it will be created in containerland as a mounted volume.